resources/views/security/login.twig

```diff
@@ -13,10 +13,13 @@
       <input id="email" type="email" name="email" value="{% if email %}{{ email | escape }}{% endif %}" class="w-full" placeholder="you@example.com">
       <label for="password" class="text-sm text-white">Password</label>
       <input id="password" type="password" name="password" class="w-full" placeholder="•••••••">
-      <p class="text-right my-4"><a href="{{ url('forgot_password') }}" class="text-white text-sm">Forgot password?</a></p>
+      <p class="text-right my-4"><a href="{{ url('forgot_password') }}" class="text-white text-sm">Forgot your password? {{ sso }}</a></p>
```

Comment:
Not sure, why are we rendering `{{ sso }}` here?

<img width="416" alt="screen shot 2018-10-03 at 09 16 55" src="https://user-images.githubusercontent.com/605483/46398494-2d0a5200-c6ed-11e8-9bab-7bb9888723a5.png">

Reply:
Removed that, must have been left over from my debugging and testing of the feature via manual testing.